tests/tests/Image/AbstractImageTest.php

```diff
@@ -597,6 +597,26 @@ public function testInOutResult($file, $in, $out)
         $thumb->save($filename);
     }
 
+    /**
+     * @dataProvider inOutResultProvider
+     * @runInSeparateProcess
```

Comment:
```suggestion
     * @runInSeparateProcess
     * @preserveGlobalState disabled
```
Maybe this helps with the PHP segfault.